spot_examples/spot_examples/batch_trajectory.py

```diff
@@ -0,0 +1,573 @@
+        """
+
+        # Claim robot.
+        self._logger.info("Claiming robot")
+        result = self._robot.command("claim")
+        if not result:
+            self._logger.error("Unable to claim robot")
+            return False
+        self._logger.info("Claimed robot")
+
+        # Power on robot.
+        self._logger.info("Powering robot on")
+        result = self._robot.command("power_on")
+        if not result:
+            self._logger.error("Unable to power on robot")
+            return False
+
+        # Stand up robot.
+        self._logger.info("Standing robot up")
+        result = self._robot.command("stand")
+        if not result:
+            self._logger.error("Robot did not stand")
+            return False
+        self._logger.info("Successfully stood up.")
+
+        # Unstow the arm.
+        self._logger.info("Unstow the arm")
+        result = self._ready_arm()
+        if not result:
+            self._logger.error("Failed to unstow the arm.")
+            return False
+        self._logger.info("Arm ready.")
+
+        # Make arm, robot and gripper follow the sampled trajectories.
+
+        start_time = time.time()
+        mobility_trajectory = self._mobility_discrete_trajectory(
+            reference_time=start_time,
+            ramp_up_time=4,
+            duration=40,
+            dt=0.05,
+            trajectory_function=mobility_continuous_trajectory,
+        )
+        arm_cartesian_trajectory = self._arm_cartesian_discrete_trajectory(
+            reference_time=start_time,
+            ramp_up_time=4,
+            duration=40,
+            dt=0.05,
+            trajectory_function=arm_cartesian_continuous_trajectory,
+        )
+
+        # Unfortunately, we cannot send at the moment gripper or arm joint
+        # trajectories with a reference time that falls before the robot time
+        # because of a limitation in the SDK.
+
+        # We leave the code here commented out.
+
+        # arm_joint_trajectory = self._arm_joint_discrete_trajectory(
+        #     reference_time=start_time,
+        #     ramp_up_time=4,
+        #     duration=40,
+        #     dt=0.05,
+        #     trajectory_function=arm_joint_continuous_trajectory,
+        # )
+        # gripper_trajectory = self._gripper_discrete_trajectory(
+        #     reference_time=start_time,
+        #     ramp_up_time=4,
+        #     duration=40,
+        #     dt=0.05,
+        #     trajectory_function=gripper_continuous_trajectory,
+        # )
+
+        command = self._build_robot_command(
+            arm_cartesian_trajectory=arm_cartesian_trajectory,
+            # arm_joint_trajectory=arm_joint_trajectory,
+            mobility_trajectory=mobility_trajectory,
+            # gripper_trajectory=gripper_trajectory,
+        )
+        action_goal = RobotCommandAction.Goal()
+        convert(command, action_goal.command)
+        self._robot_command_client.send_goal_and_wait("move_arm", goal=action_goal)
```

Comment:
nit: both the arm and the body are moving in this example, should this be more descriptive than just `move_arm`? Maybe `long_trajectory`?

Reply:
Done.